book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,261 @@
+This is a solution when you don't have the expertise or internal resources to administer a self-hosted instance of an open-source ELN solution. 
+In this case you get the best of both worlds: the benefits of professional support and the reassurance of an open solution.
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers.
+Conventional Software as a Service (SaaS) platforms are highly vertically integrated: the software development, hosting, administration, and support are package deal that you must take or leave as is.
+When using open tools each of these layers can seperated, though they do not need to be.
+This gives you the flexibility to change your provider of any of them indepentenly.
+This improves your bargaining position, but may increase procurement complexity.
+
+(rr-rdm-elns-choosing-automation)=
+### Automation and integration
+
+ELNs with APIs have the potential for integration with instruments that would permit direct deposition of data from connected instruments into ELNs or other connected research data management infrastructure.
+These sorts of integrations are especially useful in workflows at core facilities. 
+For example, bioimaging facilities can automate a substantial part of the image metadata deposition into image data management tools such as [OMERO](https://www.openmicroscopy.org/omero/).
+
+(rr-rdm-elns-standards)=
+## ELN standards and Project governance
+
+ELNs offer many advantages over paper notebooks.
+Nevertheless, ELNs come with a considerable amount of additional complexity and additional risks - so it is important to have a mitigation plan.
+The ELN space is still quite young and rapidly changing.
+As such it is important to think about how the academic community wants the ELN infrastructure to be owned and governed in the long run.
+An interesting model to consider emulating here is [moodle](https://moodle.com/), an open source electronic learning environment popular with universities.
+It offers commercial support and hosting and has a network of developers across the education sector who contribute to the core project, as well as creating custom integrations and extensions to meet their particular needs.
+This structure allows institutions to pool resources to fund the development of features that they all need and keeps this infrastructural code in common ownership, avoiding the worst lock-in issues arising from the use of proprietary solutions.
+This more open distributed model also allows individuals to work on their own custom extensions and integrations instead of being reliant on the ability and willingness of a company to implement a feature that you need.
+
+In the absense of an open standard for representing electronic lab notebook content using an open ELN solution is still no guarantee of data portability and interoperatbility.
```

Comment:
```suggestion
In the absence of an open standard for representing electronic lab notebook content using an open ELN solution is still no guarantee of data portability and interoperability.
```

I'm not sure what this sentence means: Even if there is a standard it doesn't mean that data is portable/interoperable? Or even if there is no standard this could still be the case?

Reply:
Addressed in [f64204a](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/f64204a51b90b4aad4ed831dfd8af15a5d56f531)